packages/esm-patient-common-lib/src/types/index.ts

```diff
@@ -18,3 +18,15 @@ export interface DashboardConfig extends DashboardLinkConfig {
   slot: string;
   config: DashbardGridConfig | DashboardTabConfig;
 }
+
+export enum ScreenModeTypes {
+  minimize = 'minimize',
+  maximize = 'maximize',
+  hide = 'hide',
+  reopen = 'reopen',
```

Comment:
What does `reopen` mean? Is that used anywhere? It sounds like an action, not a mode for the screen to be in.

Reply:
remove it